controllers/get-articleById-controller.js

```diff
@@ -0,0 +1,10 @@
+const { selectArticleById } = require("../models/get-articleById.model");
+exports.getArticleById = (req, res, next) => {
+    const { article_id } = req.params;
+    selectArticleById(article_id).then((article) => {
+        res.status(200).send({ article });
```

Comment:
It's ok to separate controllers into their own files but you don't have to separate every single one. You could have a controllers file for articles where any articles controllers live and then one for users etc :)